code/modules/mob/living/carbon/species.dm

```diff
@@ -646,6 +646,26 @@ var/global/list/whitelisted_species = list("Human")
 					You are particularly allergic to water, which acts like acid to you, but the inverse is so for acid, so you're fun at parties.<br>\
 					You're not as good in a fist fight as a regular baseline human, but you make up for this by bullying them from afar by talking directly into peoples minds."
 
+/datum/species/grey/handle_post_spawn(var/mob/living/carbon/human/H)
```

Comment:
why do we even take this value as an if the only valid input for it is `myhuman`? literally any other value will exit, so just take out the arg entirely and have it do its operations on `myhuman` instead of `H`